climpred/tests/test_bootstrap.py

```diff
@@ -331,3 +333,25 @@ def test_bootstrap_hindcast_raises_error(
             resample_dim='init',
             alignment='same_verifs',
         )
+
+
+def test_resample_1_size(PM_da_initialized_1d):
```

Comment:
For example, this works for me without the if statement to catch if `0` is in the resampled dim. The two functions return the final dimensions in different orders. Maybe this has something to do with the problem you were having?

Reply:
for me too